frontend/storybook/src/components/entry/admin/images/meta/pages/three.ts

```diff
@@ -0,0 +1,88 @@
+import "@elements/entry/admin/images/meta/page";
+import {LeftLabel} from "~/components/entry/admin/images/meta/sections/image-label-left";
+import "@elements/core/cards/blue";
+import "@elements/core/titles/variants/underlined-title";
+import "@elements/core/titles/variants/plain-black-list"; 
+import "@elements/core/titles/ji";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+
+
+export default {
+  title: 'Entry/Admin/Images/Meta/Pages',
+}
+
+const STR_CATEGORIES = "Categories Summary";
```

Comment:
All the static parts should be in the element. The component should *only* need to set strings for the dynamic parts.